src/functionlenses.jl

```diff
@@ -18,6 +18,13 @@ function set(obj, ::typeof(only), val)
     set(obj, first, val)
 end
 
+function set(obj, o::Base.Fix2{typeof(getfield)}, val)
+    nt = getfields(obj)
+    nt_new = @set nt[o.x] = val
+    args = Tuple(nt_new) # old julia inference prefers if we wrap in Tuple
```

Comment:
I think this wrapping is needed only for julia < 1.3 which Accessors.jl does not support anyway.